hack/deploy_cincinnati.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+# The script serves as an example of how Cincinnati can be run on an OpenShift cluster. 
+
+# Create a new namespace/project for Cincinnati
+oc new-project cincinnati
+
+# Create a dummy secret as a workaround to not having real secrets
+oc create secret generic cincinnati-credentials --from-literal=""
+
+# Apply oc template
+oc new-app -f dist/openshift/cincinnati.yaml \
+  -p IMAGE_TAG="90efacd" \
+  -p GB_PAUSE_SECS=300 \
+  -p GB_PLUGIN_SETTINGS="$(cat <<-EOF
+      [[plugin_settings]]
+      name = "release-scrape-dockerv2"
+      repository = "openshift-release-dev/ocp-release"
+      fetch_concurrency = 16
+
+      [[plugin_settings]]
+      name = "github-secondary-metadata-scrape"
+      github_org = "openshift"
+      github_repo = "cincinnati-graph-data"
+      reference_branch = "master"
+      output_directory = "/tmp/cincinnati-graph-data"
+
+      [[plugin_settings]]
+      name = "openshift-secondary-metadata-parse"
+      data_directory = "/tmp/cincinnati-graph-data"
```

Comment:
@steveeJ did something recently about making one of these directory configs unnecessary.

Reply:
That's right. The "github-secondary-metadata-scrape" plugin stores the data directory in the IO parameters which is passed to the next plugin. The "openshift-secondary-metadata-parse" plugin prefers that path in the IO parameters over the `data_directory` config setting.

Reply:
@steveeJ So I can remove the section right. 
```
 [[plugin_settings]]
      name = "openshift-secondary-metadata-parse"
      data_directory = "/tmp/cincinnati-graph-data"
```